Source/JavaScriptCore/dfg/DFGSpeculativeJIT.cpp

```diff
@@ -4242,7 +4242,7 @@ void SpeculativeJIT::compileGetPrivateNameById(Node* node)
 
         JITCompiler::Jump notCell = m_jit.branchIfNotCell(baseRegs);
 
-        cachedGetById(node->origin.semantic, baseRegs, resultRegs, stubInfoGPR, scratchGPR, node->cacheableIdentifier(), notCell, NeedToSpill, AccessType::GetPrivateName);
+        cachedGetById(node->origin.semantic, baseRegs, resultRegs, stubInfoGPR, scratchGPR, node->cacheableIdentifier(), notCell, NeedToSpill, AccessType::GetPrivateNameById);
 
```

Comment:
https://github.com/WebKit/WebKit/pull/3088#discussion_r974834669
> And I think FTL change is also required.

You need FTL change too.

Reply:
updated `compileGetPrivateNameById` to use `AccessType::GetPrivateNameById`.